src/pages/careers/locations/ciudad/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import { ciudad } from '../../_model/_locations/_locations'
+import { NewLocationLayout } from '../_location-layout-new-offices'
+import { SEO } from 'components/containers'
+import Layout from 'components/layout/layout'
+import { localize, WithIntl } from 'components/localization'
+
+const query = graphql`
+    query {
+        ciudad: file(relativePath: { eq: "careers/ciudad.png" }) {
+            ...heroImage
+        }
+        living_in_ciudad: file(relativePath: { eq: "careers/living_in_ciudad.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const CiudadDelEste = () => {
+    const images = useStaticQuery(query)
+
+    return (
+        <Layout type="careers">
+            {/* <SEO
```

Comment:
please uncomment this block if it's needed, otherwise remove it :pray:

Reply:
Waiting for SEO to be ready and will use this piece of code. Should I remove it until it's ready?

Reply:
Oh I see, ok then please add a comment or a todo so we know why this is commented right now.